sympy/functions/special/delta_functions.py

```diff
@@ -67,28 +67,30 @@ def eval(cls, arg, k=0):
         if arg.is_positive or arg.is_negative:
             return S.Zero
 
-    def simplify(self, x):
-        """simplify(self, x)
+    def _eval_expand_DiracDelta(self, **hints):
+        """expand_DiracDelta(self, **hints)
```

Comment:
There is no need to repeat the call signature here. Python's help() function and IPython scrape this using the inspect module.